### PR TITLE
Fix/login page

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: public/404.jsp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <![endif]-->
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title"><img src="/img/delft3d-gt.svg" alt="logo" /> 404 - Resource not found
+      </h3>
+    </div>
+    <div class="panel-body">
+      <h4>Looking for NetCDF Files?</h4>
+      <p>If you just clicked on the "Thredds Data Server" button and you are looking at this page, it might be that the
```
Comment (SiggyF) [inline public/404.jsp:37]: Why this message?

File: public/404.jsp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        files are simply not synchronised yet: please try again in a minute or so.</p>
+    </div>
+  </div>
+
+
+</body>
+
+</html>
```
Comment (SiggyF) [inline public/404.jsp:45]: eol

Comment (SiggyF) [reply public/404.jsp:45]: (check your editor so it always adds an  end of line  character at  the end)

File: public/complete.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html class="no-js" lang="">
```
Comment (SiggyF) [inline public/complete.html:2]: use a template?

File: src/App.vue [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="app">
+  <div class="app">
```
Comment (SiggyF) [inline src/App.vue:2]: do you have multiple apps?

File: src/assets/main.scss [hunk 5]
```diff
@@ -62,21 +62,40 @@ dt {
...
 
 .login-body {
-  img {
-    height: 20px;
-  }
+  display: flex;
 
   &::after {
-    background: url('/static/images/delta.jpg') no-repeat center center fixed;
+    background: url('/img/delta.jpg') no-repeat center center fixed;
```
Comment (SiggyF) [inline src/assets/main.scss:98]: is this optimized?

File: vue.config.js [hunk 6]
```diff
@@ -5,6 +5,7 @@ const argv = require('yargs').argv
 const apiServer = argv.apiServer || 'http://dl-ng004.xtr.deltares.nl'
 
 module.exports = {
+  filenameHashing: false,
```
Comment (SiggyF) [inline vue.config.js:8]: why?

**Review by SiggyF — APPROVED** [review]
Looks good

File: vue.config.js (hunk 6 above)
Comment (CindyvdVries) [reply vue.config.js:8]: references at server

**Pull request merged**